server/chat.js

```diff
@@ -1948,6 +1948,8 @@ Chat.filterWords = {};
 Chat.monitors = {};
 /** @type {Map<string, string>} */
 Chat.namefilterwhitelist = new Map();
+/** @type {Map<string, string>} */
```

Comment:
This should probably document what the key and value are.

Reply:
```suggestion
/**
 * Inappropriate userid : forcerenaming staff member's userid
 * @type {Map<string, string>}
 */
```

Reply:
Also I never thought we should be blacklisting userids. Sometimes we want to forcerename people for e.g. having swastika emoji in their name, but there's nothing wrong with the userid.

Reply:
Hence why the only thing this does is notify staff

Reply:
Point of order: Don't resolve conversations after replying to them, you make it so I can't see it. Just reply; I'll resolve it once I've seen it.